ibis/backends/duckdb/tests/test_register.py

```diff
@@ -110,7 +111,9 @@ def test_read_geo_from_url(con, monkeypatch):
     loaded_exts = []
     monkeypatch.setattr(con, "_load_extensions", lambda x, **_: loaded_exts.extend(x))
 
-    with pytest.raises((duckdb.IOException, duckdb.CatalogException)):
+    with pytest.raises(
+        (duckdb.IOException, duckdb.CatalogException, duckdb.NotImplementedException)
```

Comment:
@cpcloud is this the reason you added `duckdb.NotImplementedException`? 

```python-traceback
E       duckdb.duckdb.NotImplementedException: Not implemented Error: HTTPFileSystem: DirectoryExists is not implemented!

../__init__.py:104: NotImplementedException
```

I ran into this while still in 0.9.2 and notice that it gets fixed if you force install `httpfs`. Did you saw these being raises in CI, or locally? I'd suspect we don't need that on CI, as I would imagine that extensions are being installed fresh in every ci run.

Reply:
I saw these locally. We should really avoid as much divergence between CI and local as possible, so I'll probably install the httpfs extension and then report this as a bug upstream.